### ISSUE TITLE
Need for an up to date example

### ISSUE THREAD

**Issue by BeatWolf** [MartinThoma/clana#17]
Hi, your tool looks exactly like what i need, but the documentation is currently not up to date and hard to follow.

I do not what to mix multiple issues in this issue (but you can find multiple issues with the provided examples, like clana metrics not actually existing). What i would want is to provide clana with a CSV file that has two columns, actual and prediction, and an optional labels file.

Now i understand that's not currently supported. So to my actually request:

Can an example be made that can be follow from start to finish, with the actual contents of the csv files? clana get-cm takes gr and predictions csv files as input, but it is unclear to me why those need to have two columns. Can an example of a file with two columns be provided, with an explanation what the columns are?

Thank you for your great work

**Comment by MartinThoma** [comment]
Hi BeatWolf,

Thank you for the feedback. Sorry for not responding directly.

To give you a bit of context: clana was a side-project of my masters thesis 4 years ago. I had some plans with it (like the metrics module) but never followed them. I should remove those draft-parts of the code. I'll try to do it tomorrow :-)

**Comment by MartinThoma** [comment]
I'll make sure that the example is easier to follow. I'll ping you once it's done. It would be amazing if you could then check again :pray:  If you have any difficulties whatsoever after my update, please let me know. Don't spend time trying to figure it out. It should be easy to use. I'll make a step-by-step guide and you can just tell me where you're stuck
